packages/backend/discovery/amarok/ethereum/diffHistory.md

```diff
@@ -1,3 +1,35 @@
+Generated with discovered.json: 0x7a16be35e3c026ec69a014a4fd9c1fe2ad4ebb8a
+
+# Diff at Tue, 20 Aug 2024 07:24:21 GMT:
+
+- author: Radina Talanova (<nt.radina@gmail.com>)
+- comparing to: main@5417c4717b5cefeed17cd8419a7eb2dda22d4206 block: 20317972
+- current block number: 20568320
+
+## Description
+
+The optimistic mode is deactivated on Ethereum by a watcher (0x56dD71fffD089EdAdbA8eCdaaDb94269713f8f4d) and "slow" mode is active. All roots must now be passed via canonical rollup AMBs. On their bridge UI they have this message: "Due to adjustments with routers and liquidity pools this month, transaction processing on several paths may be slower than usual."
```

Comment:
might be worth it adding it as discovery description

Reply:
Added